changelog/fragments/cli-webhooks-fix.yaml

```diff
@@ -0,0 +1,17 @@
+# entries is a list of entries to include in
```

Comment:
All of these entries can go in the same fragment file. The root-level `entries` field in the fragment file is a list of entries.